### PR TITLE
Remove old FSF stuff

### PR DESCRIPTION
**Pull request by alarixnia**

### CODE REVIEW

File: class/50-host-classes [hunk 1]
```diff
@@ -69,27 +66,6 @@ case $HOSTNAME in
...
-    freestation7)
-        echo "FAIBASE DISKLESS XORG TRISQUEL WORKSTATION FSF SYSADMIN MV"
-        ;;
-    freestation06)
-        echo "FAIBASE DISKLESS XORG TRISQUEL WORKSTATION FSF SYSADMIN JAPANESE"
-        ;;
-    freestation*)
-        echo "FAIBASE DISKLESS XORG TRISQUEL WORKSTATION FSF"
-        ;;
-    zoneminder*)
```
Comment (e-lisa) [inline class/50-host-classes:84]: Would like to keep zoneminder!

File: class/50-host-classes [hunk 2]
```diff
@@ -99,15 +75,6 @@ case $HOSTNAME in
     mysqlserver*)
         echo "FAIBASE KVMGUEST SYSADMIN MYSQLD"
         ;;
-    streamstation*)
```
Comment (e-lisa) [inline class/50-host-classes:102]: would like to keep streaming related ones. But please remove the FSF class from them all. and remove TRISQUEL

File: class/50-host-classes [hunk 3]
```diff
@@ -99,15 +75,6 @@ case $HOSTNAME in
...
-        echo "FAIBASE $GRUB SYSADMIN STREAM XORG TRISQUEL LPCONF"
-	;;
-    streamtop*)
```
Comment (e-lisa) [inline class/50-host-classes:105]: Keep but remove TRISQUEL/FSF

File: class/streamtop3.var [hunk 4]
```diff
@@ -1,6 +0,0 @@
-STREAM_RTSP_IP=192.168.48.4:554
```
Comment (e-lisa) [inline class/streamtop3.var:1]: I am okay removing this file -- but maybe leave an example one for later use?
This is for Hypatia at conferences.

**Review by e-lisa — CHANGES_REQUESTED** [review]

**Comment by alarixnia** [comment]
gonna redo this

**Pull request closed without merging**